code/game/objects/items/draftingchalk.dm

```diff
@@ -0,0 +1,120 @@
+/obj/effect/decal/cleanable/draftingchalk
+	name = "chalk marking"
+	desc = "A line drawn in chalk."
+	icon = 'icons/obj/smooth/chalkline-smooth.dmi'
+	icon_state = "preview"
+	color = "#FFFFFF"
+	layer = 2.1
+	anchored = TRUE
+	smooth = SMOOTH_TRUE
+
+/obj/effect/decal/cleanable/draftingchalk/Initialize(mapload)
+	. = ..()
+	if (mapload)
+		queue_smooth(src)
+		for (var/obj/effect/decal/cleanable/draftingchalk/C in orange(1, src))
+			queue_smooth(C)
```

Comment:
Isn't every chalk already initialized with this...? Ergo, won't they already queue themselves for smoothing in line 14?

Reply:
If you don't do this chalk won't smooth with itself every time a chalk nearby is added or updates.

It looks really ugly.

Reply:
No, it's only required for the non-`mapload` side of the if. During map init it's redundant as this is going to be called on every chalk at around the same time anyways.